src/tools/spark-sstfile-generator/build.sbt

```diff
@@ -0,0 +1,86 @@
+import Commons._
+
+organization := "com.vesoft"
+// compatible with spark 1.6.0
+scalaVersion := "2.10.5"
+
+// compatible with spark 1.6.2
+// scalaVersion := "2.11.8"
+
+name := "nebula-spark-sstfile-generator"
+version := "0.1.0"
+
+test in assembly in ThisBuild := {}
+
+//not include src and doc when dist
+sources in(Compile, doc) in ThisBuild := Seq.empty
+publishArtifact in(Compile, packageDoc) in ThisBuild := false
+sources in (packageSrc) := Seq.empty
+
+libraryDependencies ++= commonDependencies
+libraryDependencies ++= Seq(
```

Comment:
remove the comment

Reply:
They are here for a reason that user's HADOOP & SPARK & Scala & SBT version may not be same of ours, which they could toggle the comment to enable what they need.